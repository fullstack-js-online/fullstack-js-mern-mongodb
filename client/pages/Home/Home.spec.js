import React from 'react'
import { mount } from 'enzyme'

import HomePage from './'

describe('The Home page', () => {
  const props = {}

  it('should match snapshot', () => {
    expect(mount(<HomePage {...props} />)).toMatchSnapshot()
  })
})
