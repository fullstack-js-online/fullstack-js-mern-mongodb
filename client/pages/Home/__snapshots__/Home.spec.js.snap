// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home page should match snapshot 1`] = `
<HomePage>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <SideEffect(NullComponent)
      defer={true}
      encodeSpecialCharacters={true}
      title="Fullstack-js Online"
      titleAttributes={Object {}}
    >
      <NullComponent
        defer={true}
        encodeSpecialCharacters={true}
        title="Fullstack-js Online"
        titleAttributes={Object {}}
      />
    </SideEffect(NullComponent)>
  </HelmetWrapper>
  <Hero>
    <div
      className="w-full h-64 flex items-center justify-center"
    >
      <h1
        className="text-5xl font-bold"
      >
        FULLSTACK-JS MERN STARTER
      </h1>
    </div>
  </Hero>
</HomePage>
`;
